addons/uh60_anvishud/initSettings.sqf

```diff
@@ -11,3 +11,30 @@
     ], 0],
     false // isGlobal
 ] call CBA_settings_fnc_init;
+
+[
+    "vtx_uh60_anvishud_settingColor",
+    "COLOR",
+    [localize LSTRING(Color), LSTRING(Color_Description)],
+    ["UH-60M", "HMD"],
+    [0.082, 0.608, 0.039],
+    false, // isGlobal
+    {
+      private _vehicle = vehicle player;
+      [_vehicle] call vtx_uh60_anvishud_fnc_configureHMD;
+    },
+    false
+] call CBA_settings_fnc_init;
+
+[
+    "vtx_uh60_anvishud_settingBrightness",
+    "SLIDER",
+    [localize LSTRING(Brightness), LSTRING(Brightness_Description)],
+    ["UH-60M", "HMD"],
+    [0, 1, 0.6, 0, true],
```

Comment:
Should probably be 2 decimal places instead of 0. Not sure what `true` is.

Reply:
Good catch on the decimals. The `true` expresses the value as a percentage in options (it's poorly documented, can't remember where I saw this originally)